src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1825,6 +1825,31 @@ public static void DCS_RecursiveCollection()
         });
     }
 
+    [Fact]
```

Comment:
Can you create a CultureInfo object on the full framework, access each of it's public properties to make sure anything lazily created is created, then serialize it. Take the output of that serialization and make sure it can be deserialized in a test here.